rerun_py/rerun_sdk/rerun/components/tensor_data.py

```diff
@@ -5,17 +5,25 @@
 
 from __future__ import annotations
 
+from typing import Any
+
+from attrs import define
+
 from .. import datatypes
 from .._baseclasses import ComponentBatchMixin
 
 __all__ = ["TensorData", "TensorDataBatch", "TensorDataType"]
 
 
+@define(init=False)
 class TensorData(datatypes.TensorData):
-    # You can define your own __init__ function as a member of TensorDataExt in tensor_data_ext.py
+    def __init__(self: Any, shape: datatypes.TensorDimensionArrayLike, buffer: datatypes.TensorBufferLike):
```

Comment:
This is is not correct. Previously delegating components would dispatch to the constructor of the Datatype, which, importantly, would in turn dispatch to the extension type.

In this case it meant that `components.TensorData` would use the constructor override from `datatypes.TensorDataExt` automatically if it didn't provide its own TensorDataExt.

Reply:
ah, so for delegating ones we shouldn't call the attribute init but the regular init then!

Reply:
.. or would leaving it out completely work? That be best ofc!

Reply:
yes. leaving it out should be the right thing to do + not putting `@define`
This should make quite a few things easier in the codegen actually!

Reply:
I was surprised the unit-tests didn't catch this but apparently we were only testing the datatype and not the component: https://github.com/rerun-io/rerun/pull/3486